community/inetutils-syslogd/APKBUILD

```diff
@@ -47,53 +45,39 @@ build() {
 		--disable-ping --disable-ping6 \
 		--disable-logger --enable-syslogd \
 		--disable-inetd --disable-whois --disable-dnsdomainname \
-		--disable-uucpd --disable-ifconfig --disable-traceroute \
-		|| return 1
-        make || return 1
+		--disable-uucpd --disable-ifconfig --disable-traceroute
+	make
 }
 
 check() {
-	cd "$srcdir/${_pkgname}-${pkgver}"
 	make check
 }
 
 package() {
 	cd "$_builddir"
```

Comment:
```suggestion
```

Reply:
@maxice8 - yes looks ok after this change is done